montepy/data_inputs/material.py

```diff
@@ -692,16 +692,16 @@ def add_nuclide(self, nuclide: Union[Nuclide, str, int], fraction: float):
             nuclide = Nuclide(nuclide)
         self.append((nuclide, fraction))
 
-    def contains(
+    def contains_all(
         self,
-        nuclide: Union[Nuclide, Nucleus, Element, str, int],
-        *args: Union[Nuclide, Nucleus, Element, str, int],
+        *nuclides: Union[Nuclide, Nucleus, Element, str, int],
```

Comment:
*args are necessarily a tuple:

```python
Tuple[Union[Nuclide, Nucleus, Element, str, int], ...]
```

This type hint is long and frequently reused. Store it as a variable and reuse it.

Reply:
perhaps somthing like `T_Nuclidelike`